### PR TITLE
Recommendation discovery - People who read this also read - initial test variants.

### PR DESCRIPTION
**Pull request by LATaylor-guardian**
This introduces the following test variants:
- recommendations produced over 30 minutes.
- recommendations produced over 3 hours.
- recommendations produced over 24 hours.

### CODE REVIEW

**Comment by LATaylor-guardian** [comment]
Not ready to merge.

File: static/src/javascripts/projects/common/modules/onward/related.js [hunk 1]
```diff
@@ -78,12 +76,23 @@ define([
 
                 container.setAttribute('data-component', componentName);
 
-                relatedUrl = popularInTag || '/related/' + config.page.pageId + '.json';
+                if (ab.isInVariant('PeopleWhoReadThisAlsoReadVariants', 'people-who-read-this-also-read-thirty-minutes')) {
+                    relatedUrl = popularInTag || '/people-who-read/thirtyMinutes' + config.page.pageId + '.json';
```
Comment (marialivia16) [inline static/src/javascripts/projects/common/modules/onward/related.js]: Don't forget the `/` at the end of the variant name :)

**Comment by LATaylor-guardian** [comment]
@rich-nguyen Can I get a :+1: for this please? I also might need a hand tomorrow getting this out on PROD as I'm not sure of your process. Thanks.

**Comment by rich-nguyen** [comment]
:+1:

**Comment by LATaylor-guardian** [comment]
@rich-nguyen thanks. Just waiting on a change to the recommendation engine our side before we begin running the tests.

File: dev-build/conf/routes [hunk 2]
```diff
@@ -105,6 +105,7 @@ GET            /top-stories.json
 GET            /top-stories/trails.json                                                                                          controllers.TopStoriesController.renderTrails()
 GET            /related/*path.json                                                                                               controllers.RelatedController.render(path)
 GET            /related/*path                                                                                                    controllers.RelatedController.renderHtml(path)
+GET            /people-who-read/:testVariant/*path.json                                                                          controllers.RelatedController.renderPeopleWhoRead(path, testVariant)
```
Comment (NathanielBennett) [inline dev-build/conf/routes]: Have you added this to the onwards routes as well?
(Or are you just running it locally)

Comment (LATaylor-guardian) [inline dev-build/conf/routes]: I'm not sure what that means? This path will only be accessed by the javascript client side on frontend.

Comment (rich-nguyen) [inline dev-build/conf/routes]: good spot @NathanielBennett the `onwards/conf/routes` file needs updating too.

Comment (LATaylor-guardian) [inline dev-build/conf/routes]: Updated.

**Pull request merged**